tests/golden/defaults/minio/minio/minio/10_minio_helmchart/minio/templates/configmap.yaml

```diff
@@ -4,7 +4,7 @@ data:
     #!/bin/sh
     set -e ; # Have script exit in the event of a failed command.
     MC_CONFIG_DIR="/etc/minio/mc/"
-    MC="/usr/bin/mc --config-dir ${MC_CONFIG_DIR}"
+    MC="/usr/bin/mc --insecure --config-dir ${MC_CONFIG_DIR}"
```

Comment:
Does this cause a problem?

Reply:
No. This just disables SSL certificate verification.